### PR TITLE
Never try to copy staging into itself. Bug#1069671

### PR DESCRIPTION
**Pull request by ancorgs**
Verified manually that it fixes https://bugzilla.suse.com/show_bug.cgi?id=1069671

### CODE REVIEW

File: test/y2storage/storage_manager_test.rb [hunk 1]
```diff
@@ -102,6 +102,33 @@ def devicegraph_from(file_name)
       manager.staging = new_graph
       expect(manager.proposal).to be_nil
     end
+
+    context "when trying to assign staging to itself" do
+      # It used to be in bug basically running staging.copy(staging), which
```
Comment (wfeldt) [inline test/y2storage/storage_manager_test.rb]: > It used to be in bug basically running staging.copy(staging)

Could you rephrase this? It's not very clear what is meant.

Comment (ancorgs) [reply test/y2storage/storage_manager_test.rb]: What about: "In the past this tried to copy staging into itself, i.e. staging.copy(staging), which caused it to become a completely empty devicegraph"

Comment (wfeldt) [reply test/y2storage/storage_manager_test.rb]: Why not just:

In the past, copying staging into itself, i.e. staging.copy(staging), caused ...
